index.html

```diff
@@ -573,6 +574,335 @@ <h3>Framing</h3>
+      that match. In this case, we'll frame the Library and Book objects
+      on values only in latinized greek (`en-Latn`):</p>
+    <pre id="library-frame-with-language-matching"
+         class="example frame nohighlight" data-transform="updateExample"
+         data-content-type="application/ld+json;profile=http://www.w3.org/ns/json-ld#framed"
+         data-frame-for="Multilingual library objects"
+         title="Library frame with language matching">
+    <!--
+    {
+      "@context": {"@vocab": "http://example.org/"},
+      "location": {"@value": {}, "@language": "en-Latn"},
+      "contains": {
+        "creator": {"@value": {}, "@language": "en-Latn"},
+        "title": {"@value": {}, "@language": "en-Latn"},
+        "contains": {
+          "title": "The Introduction"
+        }
+      }
+    }
+    -->
+    </pre>
+
+    <p>This generates the following framed results:</p>
+
+    <aside class="example ds-selector-tabs"
+           title="Framed library objects with wildcard matching">
+      <div class="selectors">
+        <a class="playground"
+           data-result-for="#multilingual-library-objects"
+           data-frame="#library-frame-with-language-matching"
+           target="_blank"></a>
+      </div>
+      <pre class="selected framed result" data-transform="updateExample"
+           data-frame="Library frame with language matching"
+           data-result-for="Multilingual library objects"
+           data-ignore> <!-- ignore as this won't match the input -->
+      <!--
+      {
+        "@context": {"@vocab": "http://example.org/"},
+        "@id": "http://example.org/library",
+        "@type": "Library",
+        "location": {"@value": "Athína", "@language": "el-Latn"},
+        "contains": {
+          "@id": "http://example.org/library/the-republic",
+          "@type": "Book",
+          "creator": {"@value": "Plátōn", "@language": "el-Latn"},
+          "title": {"@value": "Res Publica", "@language": "el-Latn"},
+          "contains": {
+            "@id": "http://example.org/library/the-republic#introduction",
+            "@type": "Chapter",
+            "description": "An introductory chapter on The Republic.",
+            "title": "The Introduction"
+          }
+        }
+      }
+      -->
+      </pre>
+    </aside>
+  </section>
+
+  <section class="informative"><h4>Matching on `@id`</h4>
+    <p>Frames can be matched if they match a specific
+      identifier (`@id`).</p>
+  </section>
+
+  <section class="informative"><h4>Empty Frame</h4>
+    <p>An empty frame matches any node object, even if those
+      objects are embedded elswhere, causing them to be serialized at the top level.</p>
+  </section>
+
+  <section class="informative"><h4>Matching with default values</h4>
+    <p>Default values can be used to match node objects, where a pattern
+      exists to match some other property, and no value exists for a property
+      having a default value.</p>
+  </section>
```

Comment:
This section looks redundant with §2.2 Default context.

Reply:
No, but maybe it should be moved. There are tests where frames match based on default values, vs just adding default values to otherwise matched frames.

Still some empty sections needing tests, which may be locked up on a powered-off computer I won't see until power comes back, hopefully later today.

Reply:
I removed it, as I couldn't find a matching test.